src/scripting/triggers.cpp

```diff
@@ -1413,7 +1413,7 @@ TRIGGER_FUNCTION(tf_accepted_culture) {
 	auto is_accepted = ve::apply(
```

Comment:
here is_accepted can be loaded directly, without apply, by simply doing nation_get_accepted_cultures(to_nation(primary_slot), trigger::payload(tval[1]).cul_id)